documentation/performance/sizing-feeding.html

```diff
@@ -247,3 +247,52 @@ <h3 id="cluster-out-of-sync">Cluster out of sync</h3>
 vds.idealstate.merge_bucket.done_failed
 </pre>
 </p>
+
+
+<h3 id="summary-store">Summary store</h3>
```

Comment:
@kkraune We should call this section "Document Store".

Reply:
agreed! wil fix